### PR TITLE
Add azure event hubs pub/sub how to

### LINKED ISSUE #559: Add Azure Event Hubs How To for Pub/Sub
Write setup document for Azure Event Hubs in Pub/Sub.

### PR DESCRIPTION
**Pull request by yaron2**
Closes #559

### CODE REVIEW

File: howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    value: <REPLACE-WITH-CONTAINER-NAME > # Required.
+```
+
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here](../../concepts/secrets/README.md)
+
+## Apply the configuration
+
+### In Kubernetes
+
+To apply the Azure Service Bus pub/sub to Kubernetes, use the `kubectl` CLI:
```
Comment (amanbha) [inline howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md:37]: Azure Event Hubs

File: howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+```bash
+kubectl apply -f eventhubs.yaml
+```
+
+### Running locally
+
+The Dapr CLI will automatically create a directory named `components` in your current working directory with a Redis component.
+To use Azure Service Bus, replace the contents of `pubsub.yaml` (or `messagebus.yaml` for Dapr < 0.6.0) file with the contents of `eventhubs.yaml` above (Don't change the filename).
```
Comment (amanbha) [inline howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md:46]: Event Hubs

File: howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md (hunk 1 above)
Comment (yaron2) [reply howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md]: Done

File: howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md (hunk 2 above)
Comment (yaron2) [reply howto/setup-pub-sub-message-broker/setup-azure-eventhubs.md]: Done

**Pull request merged**